extensions/azurePublish/src/components/ResourceGroupPicker.tsx

```diff
@@ -169,7 +171,7 @@ export const ResourceGroupPicker = ({
         ariaLabel={formatMessage(
           'A resource group is a collection of resources that share the same lifecycle, permissions, and policies'
         )}
-        disabled={loading}
+        disabled={loading || disabled}
```

Comment:
Nit: We should probably get rid of the loading part and let the caller control the disabled state entirely.  I assume this is because the caller needs to disable this during auth even if the list of names is populated?

Reply:
This is to support editing an existing profile. At the moment, we are not allowing users to change the resource group.